ChangeLog

```diff
@@ -64,6 +64,9 @@ Features
    * Add a new function mbedtls_asn1_write_named_bitstring() to write ASN.1
      named bitstring in DER as required by RFC 5280 Appendix B.
 
+Mbed Crypto
+   * Removed two Diffie-Hellman examples which were relying on the Net module.
```

Comment:
Changelog entries are primarily for users of the library. Most users of the library don't care much about module dependencies, but they might care about the example programs. So we should tell them what to look at instead. Proposal:

> Remove the Diffie-Hellman examples which implemented a toy protocol inspired by TLS DH key exchange. For an example of how to use the DHM module, see the code that calls `mbedtls_dhm_xxx` in `ssl_tls.c` and `ssl_cli.c` in Mbed TLS.

Reply:
Fixing it. (I had removed it before the review, just forgot to push.)